content/en/templates/taxonomy-templates.md

```diff
@@ -305,7 +322,8 @@ This example will list all taxonomies and their terms, as well as all the conten
 <section>
   <ul id="all-taxonomies">
     {{ range $taxonomyname, $taxonomy := .Site.Taxonomies }}
-      <li><a href="{{ "/" | relLangURL}}{{ $taxonomyname | urlize }}">{{ $taxonomyname }}</a>
+    {{ with $.Site.GetPage "taxonomyTerm" $taxonomyname }}
```

Comment:
```
{{ with $.Site.GetPage (printf "/%s" $taxonomyname) }}
```

indentation ,etc.

Reply:
The original indentation here is a bit strange, so I made some modification.